docs/reference/auditbeat/setup-repositories.md

```diff
@@ -17,10 +17,111 @@ to sign all our packages. It is available from [https://pgp.mit.edu](https://pgp
 
 ## APT [_apt]
 
-Version 9.0.0-beta1 of Beats has not yet been released.
+To add the Beats repository for APT:
+
+1. Download and install the Public Signing Key:
+
+    ```shell
+    wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | sudo apt-key add -
+    ```
+
+2. You may need to install the `apt-transport-https` package on Debian before proceeding:
+
+    ```shell
+    sudo apt-get install apt-transport-https
+    ```
+
+3. Save the repository definition to `/etc/apt/sources.list.d/elastic-{{major-version}}.list`:
+
+    ```shell subs=true
+    echo "deb https://artifacts.elastic.co/packages/{{major-version}}/apt stable main" | sudo tee -a /etc/apt/sources.list.d/elastic-{{major-version}}.list
+    ```
+
+    :::{note}
+    The package is free to use under the Elastic license. An alternative package which contains only features that are available under the Apache 2.0 license is also available. To install it, use the following sources list:
+
+    ```shell subs=true
+    echo "deb https://artifacts.elastic.co/packages/oss-{{major-version}}/apt stable main" | sudo tee -a /etc/apt/sources.list.d/elastic-{{major-version}}.list
+    ```
+    :::
+
+    :::{warning}
+    To add the Elastic repository, make sure that you use the `echo` method shown in the example. Do not use `add-apt-repository` because it will add a `deb-src` entry, but we do not provide a source package.
+
+    If you have added the `deb-src` entry by mistake, you will see an error like the following:
+
+        `Unable to find expected entry 'main/source/Sources' in Release file (Wrong sources.list entry or malformed file)`
+
+    Simply delete the `deb-src` entry from the `/etc/apt/sources.list` file, and the installation should work as expected.
+    :::
+
+4.  Run `apt-get update`, and the repository is ready for use. For example, you can install Auditbeat by running:
+
+    ```shell
+    sudo apt-get update && sudo apt-get install auditbeat
+    ```
+
+5. To configure Auditbeat to start automatically during boot, run:
+
+    ```
+    sudo systemctl enable auditbeat
+    ```
+
+    If your system does not use `systemd` then run:
```

Comment:
With https://github.com/elastic/beats/pull/41845 I don't think we have support for non systemd anymore.  The init.d scripts aren't packaged.

Reply:
Thanks for catching that @leehinman!

In [611e703](https://github.com/elastic/beats/pull/43386/commits/611e7039cb7ce00342079c2533bd5fc2311955cd) I removed the following blocks for each Beat type, for both APT and YUM:

<img width="942" alt="Screenshot 2025-03-24 at 3 41 28 PM" src="https://github.com/user-attachments/assets/dded61e7-1d1e-4426-bc1d-bf1d1d9e82b7" />